sku.0/sys.server/compiled/game/script/item/tool/crafted_armor_customization_kit.java

```diff
@@ -127,9 +120,7 @@ else if (!isValidId(player) || !exists(player))
         {
             for (Object item : items) {
                 obj_id piece = (obj_id) item;
-                if (!isCrafted(piece)) {
-                    continue;
-                } else if (static_item.isStaticItem(piece)) {
+                if ((!isCrafted(piece) || static_item.isStaticItem(piece)) && !OVERRIDE)

```

Comment:
You should check OVERRIDE first, before the other condition.  It's the quickest check and is always required to be checked so should be first.